docs/ref/migration-operations.txt

```diff
@@ -364,15 +364,6 @@ This is generally the operation you would use to create
 custom data updates and alterations, and anything else you need access to an
 ORM and/or Python code for.
 
-If you're upgrading from South, this is basically the South pattern as an
-operation - one or two methods for forwards and backwards, with an ORM and
-schema operations available. Most of the time, you should be able to translate
-the ``orm.Model`` or ``orm["appname", "Model"]`` references from South directly
-into ``apps.get_model("appname", "Model")`` references here and leave most of
-the rest of the code unchanged for data migrations. However, ``apps`` will only
```

Comment:
I considered keeping "``apps`` will only have references to models in the current app unless migrations in other apps
are added to the migration's dependencies." as this doesn't seem to be described elsewhere, however, the statement doesn't seem to be true, unless I missed something in my testing.

Reply:
This sentence is obsolete since 1aa3e09c2043c88a760e8b73fb95dc8f1ffef50e.